incubator/command-generate/src/test/resources/io/aklivity/zilla/runtime/command/generate/internal/asyncapi/mqtt/proxy/complete/zilla.yaml

```diff
@@ -98,10 +98,12 @@ catalogs:
             required:
             - id
             - status
+            type: object
```

Comment:
This is more readable if it comes first, before `properties`.
Please apply to all 4 occurrences spanning 2 files.

Reply:
done